### PR TITLE
[WIP] Romanian translation

### PR DESCRIPTION
**Pull request by silviubogan**
- [x] Have you followed the [contributing guidelines](https://github.com/github/opensource.guide/blob/master/CONTRIBUTING.md)?
- [x] Have you explained what your changes do, and why they add value to the Guides?

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

-----

This PR is dedicated to add Romanian translations. I will try to follow the same style of quotes translation like in `de`.

-----

# Status
*All tasks not marked as "in progress" are available.*

## Translation
- [x] Initial translation
    - [x] index page
    - [x] how-to-contribute.md
        - [x] Actual content
        - [x] Quotes
    - [x] starting-a-project.md
        - [x] Actual content
        - [x] Quotes
    - [x] code-of-conduct.md
    - [x] best-practices.md
    - [x] building-community.md
    - [x] finding-users.md
    - [x] getting-paid.md
    - [x] leadership-and-governance.md
    - [x] legal.md
    - [x] metrics.md
- [x] Review all the 10 translated files
    - [x] how-to-contribute.md
    - [x] starting-a-project.md
    - [x] code-of-conduct.md
    - [x] best-practices.md
    - [x] building-community.md
    - [x] finding-users.md
    - [x] getting-paid.md
    - [x] leadership-and-governance.md
    - [x] legal.md
    - [x] metrics.md
- [ ] Other small things
    - [ ] in `how-to-contribute.md`, gaearon's quote at line 510 contains an `<em>`, it is not well displayed as different from the em-styled content around it (*)
    - [ ] in `best-practices.md`, the screenshot `/assets/images/best-practices/celery.png` is not translated (and there are many other screenshots that could somehow be translated) - what would be the best HTML to use for the translation of this screenshot and where to put it: before the original screenshot, or after?
    - [x] replace title `Învățând să spui nu` in `ro/best-practices.md` with `Învățând să spui „nu”` (Romanian quotes) and also correct the links to that section in all the translated files if the section link changes to contain "„" or "”".
    - [x] all section (#) links should have their URL translated, currently they do not point to existing parts of the current page (the Travis CI build shows these errors)
    - [x] replace in all places "a comite" with "a face un commit" because "a comite" has criminal connotation (just in how-to-contribute.md & starting-a-project.md)

## Other tasks
- [ ] solve warnings about '„' and '”' characters in the website's compilation process (the Travis CI build shows these errors)
- [ ] the "Section 1", "Section 2" etc. headers are not translated (the same situation as in other translations)
- [x] the navigation menu at the top of the pages is with empty strings in the case of Romanian locale
- [x] the drop-down with languages displays the Romanian translation as an empty string
- [x] the footer of the pages at the bottom of the page is invisible in the `ro` version of the pages and the images that link to other articles at the bottom of the pages have broken URLs
- [x] the home page of the website shows duplicate links for some of the articles: one in English, and the other one in Romanian, although the language is set to English (if I set the language to Romanian, just one link for each translated article shows up, currently with the exception of the article with translation in progress (best-practices.md))
- [x] in the website footer "friends" does not get translated in the case of the main page in Romanian, but it is translated in the Romanian article pages (I tried to rebuild the website using `/script/bootstrap` and `/script/test` and `/script/server` but I think I may have to delete the generated files before)

### CODE REVIEW

File: _data/locales/ro.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+en:
```
Comment (bartmathijssen) [inline _data/locales/ro.yml]:
```suggestion
ro:
```

Comment (bartmathijssen) [inline _data/locales/ro.yml]:
```suggestion
ro:
```

**Comment by ghaith6558** [comment]
NP

**Comment by ghaith6558** [comment]
np

**Comment by silviubogan** [comment]
The only problem that I think is really necessary to be solved before merging is the failing test.

> solve warnings about '„' and '”' characters in the website's compilation process (the Travis CI build shows these errors)

I do not know how to solve this. All I can say is that the correct Romanian double quotes (as in the `Romanian (Programmers)` keyboard layout) are `„` at the beginning and `”` at the end.

The other problems are small technical problems that I do not know how to solve.

**Comment by MikeMcQuaid** [comment]
So sorry for the delay! Thanks so much for your contribution! Without people like you submitting PRs we couldn't run this project. You rock, @silviubogan!

**Comment by komyo904** [comment]
Thank you for your contribution but it looks like you have a good number of errors to work through via the GitHub Actions CI/tests status check.

**Pull request merged**